assignment4/ques1.js

```diff
@@ -0,0 +1,21 @@
+// Write a program to remove duplicate charecters from a string ("Hello Yellow" => "Helo Yw")
+function removeDuplicate(str){
+    let s=str.split('');
+    let unique_s=s.filter((item,index)=> s.indexOf(item)==index);
```

Comment:
You can check for better way instead of applying indexOf inside filter that will increase time complexity.